Hoxtygen/talksofcode.com
# Chore add google lighthouse workflow #31

#### What does this PR do?
Add chrome lighthouse workflow to collect performance metrics

#### Description of Task to be completed?
* create `.yml` file  for lighthouse workflow
* add lighthouse configuration file
* install `@lhci/cli` to run locally
* add script to test locally

#### How should this be manually tested?
* clone the repo
* open a terminal on your PC and run git clone <repo_url>
*  cd into the project repo and run yarn install
* in the terminal run `yarn lhci:mobile` or `yarn lhci:desktop`
* click on provided link to see result

#### What are the relevant azure board stories?
[[#31](https://hoxtygen.visualstudio.com/talksofcode.com/_sprints/taskboard/talksofcode.com%20Team/talksofcode.com/Iteration%203?workitem=31)]

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/19408669/199258831-6c9d5259-8b76-44d0-8079-07476cc3d1fc.png)


#### Questions:
